src/index.ts

```diff
@@ -26,13 +26,19 @@ function loader(this: interfaces.Webpack, contents: string) {
         return;
     }
 
+    // Load files generated by previous loaders in the same chain
+    const generatedFiles = getGeneratedFiles(this);
```

Comment:
The option `usePreviousLoaderGeneratedFiles` has been implemented in the latest commit.